### PR TITLE
add react course & tutorial

### PR DESCRIPTION
**Pull request by rizaradiarivaldo**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-id.md [hunk 1]
```diff
@@ -219,7 +219,8 @@
 * [ReactJS Untuk Pemula](https://www.petanikode.com/reactjs-untuk-pemula/) - Petani Kode
```
Comment (davorpa) [inline courses/free-courses-id.md:219]: Using Prawito Hudoro's Youtube playlist where video `frwlr7uh0io` belongs to.
```suggestion
* [ReactJS Untuk Pemula](https://www.petanikode.com/reactjs-untuk-pemula/) - Petani Kode
* [ReactJS-Firebase Tutorial](https://www.youtube.com/playlist?list=PLU4DS8KR-LJ2CnIvj7tI0zoijDSgR1m9j) - Prawito Hudoro
```

File: courses/free-courses-id.md [hunk 2]
```diff
@@ -219,7 +219,8 @@
...
 * [Tutorial React JS Bahasa Indonesia](https://www.youtube.com/playlist?list=PLp6BJq2fT_g91yCNCWi_bIe-ng7S7rt6V) - Lampung JS
 * [Tutorial React Native Bahasa Indonesia (Futsal App)](https://www.youtube.com/playlist?list=PLIan8aHxsPj2NeWJew3o86bSptVPXOppa) - Wahidev Academy
-
+* [Setup Redux Thunk Pada Project React](https://www.youtube.com/watch?v=frwlr7uh0io) - prawito hudoro
+* [Ayo Kita Bahas Middleware](https://www.youtube.com/watch?v=kVV17VI9qTU) - Parsinta
```
Comment (davorpa) [inline courses/free-courses-id.md:223]: - Using Youtube playlist where `frwlr7uh0io` belongs to.
- Adding the instructor @irsyadadl as well
- Recover blank line after list

```suggestion
* [Mari Kita Belajar Basic React JS](https://www.youtube.com/playlist?list=PLRKMmwY3-5MwXT8zMPbezhDnTM3cTA5cZ) - Irsyad A. Panjaitan, Parsinta

```

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to apply some changes. When done, request for a review again using GitHub :arrows_clockwise: button. Meanwhile, draft status can be used to work on it if you want.

File: courses/free-courses-id.md (hunk 2 above)
Comment (irsyadadl) [reply courses/free-courses-id.md:223]: Hey, thanks for adding me.

[the remark this answers is not in the document]
Comment (davorpa) [reply courses/free-courses-id.md:223]: changes proposed at https://github.com/EbookFoundation/free-programming-books/pull/6313#discussion_r739754348

File: courses/free-courses-id.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-id.md:219]: Solved at https://github.com/EbookFoundation/free-programming-books/pull/6313/commits/5e861a4e6d350e39626173986741e614947ce56f

File: courses/free-courses-id.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-id.md]: Solved at https://github.com/EbookFoundation/free-programming-books/pull/6313/commits/5e861a4e6d350e39626173986741e614947ce56f

File: courses/free-courses-id.md [hunk 3]
```diff
@@ -223,8 +223,10 @@
 * [React JS Dasar Bahasa Indonesia](https://www.youtube.com/playlist?list=PLIan8aHxsPj0XtJjWW04hN24fWXrCpLkY) - Wahidev Academy
 * [ReactJS Tutorial](https://www.youtube.com/playlist?list=PLU4DS8KR-LJ03qEsHn9zV4qdhcWtusBqb) - Prawito Hudoro
 * [ReactJS Untuk Pemula](https://www.petanikode.com/reactjs-untuk-pemula/) - Petani Kode
+* [ReactJS-Firebase Tutorial](https://www.youtube.com/playlist?list=PLU4DS8KR-LJ2CnIvj7tI0zoijDSgR1m9j) - Prawito Hudoro (YouTube)
 * [Tutorial React JS Bahasa Indonesia](https://www.youtube.com/playlist?list=PLp6BJq2fT_g91yCNCWi_bIe-ng7S7rt6V) - Lampung JS
 * [Tutorial React Native Bahasa Indonesia (Futsal App)](https://www.youtube.com/playlist?list=PLIan8aHxsPj2NeWJew3o86bSptVPXOppa) - Wahidev Academy
+* [Mari Kita Belajar Basic React JS](https://www.youtube.com/playlist?list=PLRKMmwY3-5MwXT8zMPbezhDnTM3cTA5cZ) - Irsyad A. Panjaitan, Parsinta (YouTube)
```
Comment (davorpa) [inline courses/free-courses-id.md:229]: Move to line 223 more or less as solution for linter faults. List alphabetization must be preserved.

**Review by davorpa — CHANGES_REQUESTED** [review]

File: courses/free-courses-id.md [hunk 4]
```diff
@@ -223,8 +223,10 @@
 * [React JS Dasar Bahasa Indonesia](https://www.youtube.com/playlist?list=PLIan8aHxsPj0XtJjWW04hN24fWXrCpLkY) - Wahidev Academy
 * [ReactJS Tutorial](https://www.youtube.com/playlist?list=PLU4DS8KR-LJ03qEsHn9zV4qdhcWtusBqb) - Prawito Hudoro
 * [ReactJS Untuk Pemula](https://www.petanikode.com/reactjs-untuk-pemula/) - Petani Kode
+* [ReactJS-Firebase Tutorial](https://www.youtube.com/playlist?list=PLU4DS8KR-LJ2CnIvj7tI0zoijDSgR1m9j) - Prawito Hudoro (YouTube)
```
Comment (davorpa) [inline courses/free-courses-id.md:226]: Move it to line 224 more or less to solve linter faults: https://github.com/EbookFoundation/free-programming-books/runs/4131249220?check_suite_focus=true

**Review by irsyadadl — APPROVED** [review]

**Review by irsyadadl — APPROVED** [review]

**Review by irsyadadl — APPROVED** [review]

**Review by irsyadadl — APPROVED** [review]

**Review by irsyadadl — APPROVED** [review]

**Pull request merged**